plugins/Feedback/lang/de.json

```diff
@@ -42,6 +41,7 @@
         "PurgeOldData": "Ich möchte einige alte Daten entfernen",
         "SearchHelpResources": "matomo.org Hilfequellen durchsuchen",
         "PopularHelpTopics": "Beliebte Hilfethemen",
-        "ReferMatomo": "Matomo empfehlen"
+        "ReferMatomo": "Matomo empfehlen",
+        "ReferBannerTitle": "Vertrauen Sie Matomo?"
```

Comment:
That's a bit weird translation for `Like Matomo?`

@sgiehl Do you have a better idea? "Gefällt Ihnen Matomo"? maybe?

Reply:
Guess that would be a better translation.

Reply:
:+1: I updated the translation on transifex

Reply:
Ok. will trigger a new translation update

Reply:
Okay, I seem to have mixed up the translation keys. There is "Like Matomo?" => "Mögen Sie Matomo?" and "Believe in Matomo?" => ?
I don't really have a fitting translation for this key

Reply:
How about "Konnte Matomo Sie überzeugen?"

Reply:
@sgiehl I like that one and updated it on transifex.
Thansk